public/walkthroughs/1A/walkthrough.adoc

```diff
@@ -342,7 +342,7 @@ After setting up the integration between the Node.js and Spring Boot application
 :spring-url: http://spring-boot-rest-http-crud-spring-app.apps.city.openshiftworkshop.com/
 :fuse-url: https://eval.apps.city.openshiftworkshop.com/
 
-. Navigate to the link:http://{node-js-url}[Node.js app, window="_blank"].
+. Navigate to the link:{route-frontend-host}[Node.js app, window="_blank"].
```

Comment:
link renders as `http://localhost:3006/tutorial/1A/task/%7Broute-frontend-host%7D` :(

Reply:
looks like the namespace doesn't get created

```  "message": "ProjectRequest.project.openshift.io \"evals22-example-com-1A\" is invalid: metadata.name: Invalid value: \"evals22-example-com-1A\": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')",
```